src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1210,77 +1211,26 @@ internal static FileSystemCmdletProviderEncoding GetEncoding(string path)
             string initialBytesAsAscii = System.Text.Encoding.ASCII.GetString(initialBytes, 0, bytesRead);
             if (initialBytesAsAscii.IndexOfAny(nonPrintableCharacters) >= 0)
             {
-                return FileSystemCmdletProviderEncoding.Byte;
+                return Encoding.Unicode;
```

Comment:
Returning `Encoding.Unicode` doesn't seem right. It looks we should deal with bytes in this case.

Reply:
There is no byte encoding, and FileSystemCmdletProviderEncoding.Byte wound up being converted to a unicode encoding in GetEncodingFromEnum (now removed) so there's actually no change here.